### PR TITLE
Update README.md for new simpler development cycle

### PR DESCRIPTION
**Pull request by michaelsembwever**
Remove `content/` folder as it will be served from asf-staging and asf-site branches.

 ref: https://lists.apache.org/thread.html/r83ae2d46cf247163316c2469bf7ed9cce1e8df67b94d2fca813fd548%40%3Cdev.cassandra.apache.org%3E

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+
+Development Cycle
+-----------------
+
+Making changes to the website is done with the following steps.
+
+1. Test changes locally
+2. Commit changes in `src/` to a fork and branch, and create a pull request
+3. Get the pull request reviewed and merged to `master`
+4. Test on https://cassandra.staged.apache.org/ (wait til [ci-cassandra.apache.org](https://ci-cassandra.apache.org/job/cassandra-website/) has deployed it)
```
Comment (ossarga) [inline README.md:12]: Nitpick - reword slightly to say:
> Preview the rendered site on ...

File: README.md [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+5. Merge `asf-staging` to `asf-site`
+6. Test on https://cassandra.apache.org/
```
Comment (ossarga) [inline README.md:14]: Similar to previous comment. May be say something like:
> View the rendered site on ...

File: README.md [hunk 3]
```diff
@@ -1,10 +1,23 @@
...
 
-Building the Apache Cassandra website takes a number of steps. To make things easier we have provided a Docker container which can build the full website in two simple commands and have it ready to commit via git. If you are interested in the process head over to the [README](./src/README) in _src_ directory.
 
-Prerequisite
-------------
+To test changes before committing, it is required to build the website locally. Building the Apache Cassandra website takes a number of steps. To make things easier we have provided a Docker container which can build the full website in two simple commands and have it ready to commit via git. If you are interested in the process head over to the [README](./src/README) in _src_ directory.
```
Comment (ossarga) [inline README.md:17]: Might need to reword the first sentence. Specifically add a few words to the part after the comma. Feel free to use any of the following variations:

> To test changes before committing, it is a requirement that you build the website locally.

OR

> To test changes before committing, it is required that you build the website locally.

OR (completely reword it)

> To test changes before committing, you will need to build the website locally.

**Comment by michaelsembwever** [comment]
Great feedback @ossarga . All suggestions applied.

**Pull request merged**